### DISCUSSION TITLE
magic-pdf.json 中的参数是否可以使用环境变量注入的方式

### DISCUSSION THREAD

**Discussion by Tendo33** [magicpdf/Magic-PDF#647] [Ideas]
如题

**Comment by myhloli** [comment]
后面有计划增加环境变量的配置方式

**Reply by shari4us** [reply]
目前，Magic-PDF 的配置文件（[magic-pdf.json](https://shayari4us.com/attitude-shayari-in-english/)）不支持直接使用环境变量注入的方式。为了实现环境变量注入，可能需要在应用代码中动态读取并替换配置文件中的值，或者考虑通过其他配置管理工具来实现环境变量的自动替换。如果是常见的需求，建议提出 Feature Request 或查看是否有类似的功能实现。

**Comment by myhloli** [comment]
https://github.com/opendatalab/MinerU/blob/master/magic_pdf/libs/config_reader.py

现在应该是支持环境变量修改配置文件路径了
